cob_twist_controller/include/cob_twist_controller/inverse_differential_kinematics_solver.h

```diff
@@ -83,7 +84,10 @@ class InverseDifferentialKinematicsSolver
         KDL::Frame dummy;
         dummy.p = KDL::Vector(0,0,0);
         dummy.M = KDL::Rotation::Quaternion(0,0,0,0);
```

Comment:
use spaces to separate parameters...

also, I think the quaternion should be initialized with w=1.0!